mobile/src/reducers/mediaBackup.js

```diff
@@ -12,11 +12,13 @@ export const mediaBackup = (state = initialState, action) => {
     case MEDIA_UPLOAD_START:
       return { ...state, uploading: true }
     case MEDIA_UPLOAD_END:
-      return { ...state, uploading: false }
+      return { ...state, uploading: false, currentUpload: undefined }
     case IMAGE_UPLOAD_SUCCESS:
-      return { ...state, uploaded: [...state.uploaded, action.id] }
+      return { ...state, uploaded: [...state.uploaded, action.id], currentUpload: undefined }
```

Comment:
if you do that (currentUpload: undefined) for each photo, the UploadProgression component will blinking...